src/lib.rs

```diff
@@ -9,6 +9,7 @@ trait StoreImpl {
     }
     fn get<T: DeserializeOwned>(&self, key: &str) -> Result<T, Self::GetError>;
     fn set<T: Serialize>(&mut self, key: &str, value: &T) -> Result<(), Self::SetError>;
+    fn clear(&mut self) -> Result<(), Self::SetError>;
```

Comment:
I'm a little unsure about what time best approach to error handling is... Having clear return a seterror feels a bit weird, not sure if i have a better suggestion, though, just commenting in case you do ;)

Feel free to ignore otherwise.

Reply:
I had the same confusion about the return type of error at  first,but i am not sure should I add new error type,like
```
pub enum OperationError{
Clear,
xxx
}
```
this may can provide the error type like `len()` or `get_keys()` in the future.
but at the same time i am not sure if i should add this error so i add setError simply